packages/paste-core/utilities/flex/src/index.tsx

```diff
@@ -0,0 +1,128 @@
+import * as React from 'react';
+import * as PropTypes from 'prop-types';
+import {Box} from '@twilio-paste/box';
+import {FlexboxProps} from '@twilio-paste/types';
+
+export type displayOptions = 'flex' | 'inline-flex';
+export type vAlignOptions = 'top' | 'center' | 'bottom' | 'stretch';
+export type hAlignOptions = 'left' | 'center' | 'right' | 'around' | 'between';
+
+export interface FlexProps {
+  display?: displayOptions;
```

Comment:
`import {ResponsiveValue} from 'styled-system';`
```
ResponsiveValue<DisplayOptions>;
```
similar treatment to the other types here